Doc/faq/programming.rst

```diff
@@ -273,7 +278,7 @@ main.py::
    import mod
    print(config.x)
 
-Note that using a module is also the basis for implementing the Singleton design
+Note that using a module is also the basis for implementing the singleton design
```

Comment:
This change was not useful: classic design patterns are commonly title cased.
But not worth changing back now.